### PR TITLE
MSTest Playwright SDK sample

### PR DESCRIPTION
**Pull request by Baaryan**

### CODE REVIEW

File: .github/workflows/sanity_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            repo: context.repo.repo,
+            name: process.env.job_name,
+            head_sha: process.env.commit_sha,
+            status: 'completed',
+            conclusion: process.env.conclusion
+            }).catch((err) => ({status: err.status, response: err.response}));
+            console.log(`The status-check response : ${result.status} Response : ${JSON.stringify(result.response)}`)
+            if (result.status !== 201) {
+              console.log('Failed to create check run')
+            }
```
Comment (kamal-kaur04) [inline .github/workflows/sanity_workflow.yml]: new line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+*.binlog
+
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
+
+# MFractors (Xamarin productivity tool) working folder
+.mfractor/
+
+# Local History for Visual Studio
+.localhistory/
```
Comment (kamal-kaur04) [inline .gitignore]: new line

File: MSTest-Playwright-BrowserStack/.config/dotnet-tools.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "isRoot": true,
+  "tools": {
+    "browserstack-sdk": {
+      "version": "1.12.2",
+      "commands": [
+        "browserstack-sdk"
+      ]
+    }
+  }
+}
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/.config/dotnet-tools.json]: new line

File: MSTest-Playwright-BrowserStack/.config/dotnet-tools.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "browserstack-sdk": {
+      "version": "1.12.2",
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/.config/dotnet-tools.json]: Please remove this browserstack-sdk tool, not needed.
`"tools": {}`

File: MSTest-Playwright-BrowserStack/SampleTest.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                // Verify if the cart has the right item
+                StringAssert.Contains(await cartItem.InnerTextAsync(), string.Join(" ", phone));
+                IReadOnlyList<string> cartItemText = await cartItem.AllInnerTextsAsync();
+                Console.WriteLine("Cart item => " + cartItemText[0]);
+
+                Assert.AreEqual(phone[0], cartItemText[0]);
+            }
+        }
+    }
+}
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/SampleTest.cs]: new line

File: MSTest-Playwright-BrowserStack/Usings.cs [hunk 6]
```diff
@@ -0,0 +1 @@
+global using Microsoft.VisualStudio.TestTools.UnitTesting;
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/Usings.cs]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: MSTest-Playwright-BrowserStack/.config/dotnet-tools.json (hunk 4 above)
Comment (Baaryan) [reply MSTest-Playwright-BrowserStack/.config/dotnet-tools.json]: Sure

File: .github/workflows/sanity_workflow.yml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
```
Comment (kamal-kaur04) [inline .github/workflows/sanity_workflow.yml]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Review by RutvikChandla — APPROVED** [review]

**Pull request merged**